### PR TITLE
Disable coverage/warnings output when passing in dev

### PR DESCRIPTION
**Pull request by bf4**
cc @jfelchner

### CODE REVIEW

**Comment by jfelchner** [comment]
You rock my socks. 😀

**Comment by NullVoxPopuli** [comment]
ya got something failing in your tests :-)

File: .simplecov [hunk 1]
```diff
@@ -49,21 +49,32 @@ Coverage.start
 
 ## ADD SOME CUSTOM REPORTING AT EXIT
 SimpleCov.at_exit do
+  next if $! and not ($!.kind_of? SystemExit and $!.success?)
```
Comment (bf4) [inline .simplecov]: Don't print coverage on an exception

File: .travis.yml [hunk 2]
```diff
@@ -10,15 +10,14 @@ rvm:
...
-  - rbx-2
   - ruby-head
+  - rbx-2
 
 install:
   - bundle install --retry=3
 
 script:
-  - bundle exec rake
+  - env CAPTURE_STDERR=false bundle exec rake
```
Comment (bf4) [inline .travis.yml]: failures in ci easier to understand

File: .simplecov (hunk 1 above)
Comment (beauby) [inline .simplecov]: :+1: I always get annoyed by those meaningless coverage stats when tests fail with exceptions.

**Pull request merged**